src/main/java/org/broadinstitute/hellbender/utils/read/ArtificialReadUtils.java

```diff
@@ -162,40 +162,42 @@ public static GATKRead createArtificialRead(final Cigar cigar) {
         return new SAMRecordToGATKReadAdapter(createArtificialSAMRecord(cigar));
     }
 
+    public static GATKRead createArtificialRead(final Cigar cigar, final String name) {
```

Comment:
Can you implement `createArtificialRead(final String cigarString)` in terms of this one?